src/main.cpp

```diff
@@ -5642,26 +5728,26 @@ void ProcessTxBaseMsg(const CTransactionBase& txBase, CNode* pfrom)
             // Never relay transactions that we would assign a non-zero DoS
             // score for, as we expect peers to do the same with us in that
             // case.
-            int nDoS = 0;
-            if (!state.IsInvalid(nDoS) || nDoS == 0) {
+            if (!state.IsInvalid() || state.GetDoS() == 0)
```

Comment:
you may use another method `if(state.IsValid() || ..)`

Reply:
I leave it as it is since it also match the 'error' case